scripts/shadowed/unshadow-mount-points

```diff
@@ -161,7 +161,7 @@ function create_bind_mounts()
 
     ROOT_DEVICE=$(grep ' / ' /proc/mounts | cut -d ' ' -f 1)
 
-    for MOUNT_POINT in $(grep -v "^$ROOT_DEVICE" /proc/mounts | egrep -v " /(sys|proc|dev|run|mnt)" | cut -d ' ' -f 2 | grep -v '^/$')
+    for MOUNT_POINT in $(grep -v "^$ROOT_DEVICE" /proc/mounts | grep -E -v " /(sys|proc|dev|run|mnt)" | cut -d ' ' -f 2 | grep -v '^/$')
```

Comment:
Again (see above): Why prever `grep -E` over `egrep`?

Reply:
`egrep` emits a message to stderr https://lists.gnu.org/archive/html/info-gnu/2022-09/msg00001.html